src/script/view_model/content/AppLockViewModel.ts

```diff
@@ -46,28 +47,41 @@ export enum APPLOCK_STATE {
 
 const APP_LOCK_STORAGE = 'app_lock';
 
+const getInactivityTimeout = () => {
+  const teamState = container.resolve(TeamState);
```

Comment:
I think it would be better to move this function into the view model since the teamState exists there already. You probably would have to move connected functions as well.